xbmc/guilib/GUIFontManager.cpp

```diff
@@ -438,10 +467,54 @@ void GUIFontManager::LoadFonts(const TiXmlNode* fontNode)
 
     if (!fontName.empty() && URIUtils::HasExtension(fileName, ".ttf"))
     {
-      //! @todo Why do we tolower() this shit?
       std::string strFontFileName = fileName;
+      std::string strFontFileNameFallback;
+      if (fontSet)
+      {
+        const std::shared_ptr<CSettings> settings =
+            CServiceBroker::GetSettingsComponent()->GetSettings();
+        std::string currentLanguage = settings->GetString(CSettings::SETTING_LOCALE_LANGUAGE);
+        StringUtils::Replace(currentLanguage, "resource.language.", "");
+
+        const TiXmlElement* pChild = fontSet->FirstChildElement("font_override");
+        while (pChild)
+        {
+          const char* idAttr = pChild->Attribute("font");
+          if (idAttr)
+          {
+            auto ids = StringUtils::Split(idAttr, "|");
+            auto list = std::find(ids.begin(), ids.end(), fileName);
+            if (list == ids.end())
+              continue;
+
+            const TiXmlElement* pChildLanguage = pChild->FirstChildElement("language");
+            while (pChildLanguage)
+            {
+
+              const char* language = pChildLanguage->Attribute("id");
+              if (language && StringUtils::CompareNoCase(language, currentLanguage) == 0 &&
+                  pChildLanguage->GetText() != nullptr)
+              {
+                const char* no_fallback = pChildLanguage->Attribute("no_fallback");
+                if (!no_fallback || StringUtils::CompareNoCase(no_fallback, "true") != 0)
+                  strFontFileNameFallback = strFontFileName;
+                strFontFileName = pChildLanguage->GetText();
+                break;
+              }
+
+              pChildLanguage = pChildLanguage->NextSiblingElement("language");
+            }
+
+            break;
+          }
+          pChild = pChild->NextSiblingElement("font_override");
+        }
+      }
+
+      //! @todo Why do we tolower() this shit?
```

Comment:
Maybe improve this comment a bit?

Reply:
Yeah, was already in https://github.com/xbmc/xbmc/blob/master/xbmc/guilib/GUIFontManager.cpp#L441

Thinking of looking at the problem and removing the comment: smile:.